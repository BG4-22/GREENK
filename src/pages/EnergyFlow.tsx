import { Center } from '@chakra-ui/react';
import Energiflyt from 'assets/videos/Energyflow-video.mp4';

/**
 * Video-component to show energyflow. A centered video-element.
 */

const EnergyFlow: React.FC = () => {
    return (
        <Center margin={'75px auto'}>
            <video width="75%" autoPlay loop style={{ borderRadius: '40px' }}>
                <source src={Energiflyt} type="video/mp4"></source>
            </video>
        </Center>
    );
};

export default EnergyFlow;
