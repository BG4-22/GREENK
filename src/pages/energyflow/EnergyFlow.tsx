import { Center } from '@chakra-ui/react';
import Energiflyt from '../../assets/Energiflyt_Nidarvoll.mp4';

const EnergyFlow: React.FC = () => {
    return (
        <Center margin={'20px auto'}>
            <video width="75%" autoPlay loop style={{ borderRadius: '40px' }}>
                <source src={Energiflyt} type="video/mp4"></source>
            </video>
        </Center>
    );
};

export default EnergyFlow;
