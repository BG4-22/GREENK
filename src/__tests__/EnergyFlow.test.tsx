import { shallow } from 'enzyme';
import toJson from 'enzyme-to-json';
import React from 'react';
import EnergyFlow from '../pages/EnergyFlow';

describe('<EnergyFlow />', () => {
    const wrapper = shallow(<EnergyFlow />);
    it('renders correctly', () => {
        expect(toJson(wrapper)).toMatchSnapshot();
        expect(wrapper).toHaveLength(1);
    });

    it('contains video-element', () => {
        expect(wrapper.find(<video></video>)).toBeDefined();
    });
});
