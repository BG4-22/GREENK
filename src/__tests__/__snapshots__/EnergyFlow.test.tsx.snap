// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<EnergyFlow /> renders correctly 1`] = `
<Center
  margin="20px auto"
>
  <video
    autoPlay={true}
    loop={true}
    style={
      Object {
        "borderRadius": "40px",
      }
    }
    width="75%"
  >
    <source
      src="stub-file"
      type="video/mp4"
    />
  </video>
</Center>
`;
